ddp-workspace/projects/ddp-rarex/app.yaml

```diff
@@ -0,0 +1,20 @@
+service: rarex
+runtime: python37
+
+# This setting will not matter except to prevent default automatic-scaling which prevents disabling
+# earlier versions of this service
+basic_scaling:
+  max_instances: 1
+
+handlers:
+  - url: /(.*\.(css|eot|gz|html|ico|js|map|png|svg|ttf|woff|woff2|jpg|jpeg|gif|png|json|pdf))
+    static_files: dist/\1
+    upload: dist/(.*\.(css|eot|gz|html|ico|js|map|png|svg|ttf|woff|woff2|jpg|jpeg|gif|png|json|pdf))
+    secure: always
+    http_headers:
+      Access-Control-Allow-Origin: https://rarex-dev.us.auth0.com
```

Comment:
What's the purpose of this header? I don't see this in the `app.yaml` for other studies?

Reply:
I believe it was done to make sure that Auth0 pages could access application's assets although it should be redundant to have it here.